.netlify/build-and-move.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Build the static web site in website/build
+# bundle install     should be cached???
+bundle exec middleman build
```

Comment:
I'm sure this is fine, and I don't know what the equivalent would be if we were to use the docker container setup, but this difference between local development and netlify build gives me pause.